wire-tests-swift/no-manifest/src/main/swift/AllTypes.swift

```diff
@@ -848,6 +848,9 @@ public struct AllTypes {
 
 }
 
+extension AllTypes : ProtoExtensible {
```

Comment:
This also should be generating:
```
public var ext_opt_int32: Int32? {]
    get { storage.ext_opt_int32 }
    set { storage.ext_opt_int32 = newValue }
}
```

Reply:
Otherwise you'd be paying `dynamicMemberLookup` costs

Reply:
Will be updating.

Reply:
done